### ISSUE TITLE
Generated tag should have the same repr as the input

### ISSUE THREAD

**Issue by RReverser** [Robbepop/enum-tag#6]
I noticed from the README example that `repr(u8)` is currently not propagated to the generated tag enum. This can make it more difficult to reconcile the tagged and untagged enums with each other, especially when repr is important e.g. for FFI.

**Comment by Robbepop** [comment]
Yes, that's probably right. I happily accept pull requests if you are interested to contribute. :)

**Comment by RReverser** [comment]
As soon as I finish with my current side projects... 😅 (which is probably a while)
